HLTrigger/Egamma/plugins/HLTEgammaDoubleLegCombFilter.cc

```diff
@@ -62,90 +60,100 @@ bool HLTEgammaDoubleLegCombFilter::hltFilter(edm::Event& iEvent, const edm::Even
+    matchedCands.push_back(std::make_pair(firstLegNr, matchedNr));
+    if (matchedNr >= 0)
+      matched2ndLegs.push_back(static_cast<size_t>(matchedNr));
   }
-  std::sort(matched2ndLegs.begin(),matched2ndLegs.end());
+  std::sort(matched2ndLegs.begin(), matched2ndLegs.end());
 
-  for(size_t secondLegNr=0;secondLegNr<secondLegP3s.size();secondLegNr++){
-    if(!std::binary_search(matched2ndLegs.begin(),matched2ndLegs.end(),secondLegNr)){ //wasnt matched already
-      matchedCands.push_back(std::make_pair<int,int>(-1, static_cast<int>(secondLegNr)));
+  for (size_t secondLegNr = 0; secondLegNr < secondLegP3s.size(); secondLegNr++) {
+    if (!std::binary_search(matched2ndLegs.begin(), matched2ndLegs.end(), secondLegNr)) {  //wasnt matched already
+      matchedCands.push_back(std::make_pair<int, int>(-1, static_cast<int>(secondLegNr)));
     }
   }
 }
 
 //we use position and p3 interchangably here, we only use eta/phi so its alright
-void  HLTEgammaDoubleLegCombFilter::getP3OfLegCands(const edm::Event& iEvent,const edm::EDGetTokenT<trigger::TriggerFilterObjectWithRefs>& filterToken,std::vector<math::XYZPoint>& p3s)
-{
+void HLTEgammaDoubleLegCombFilter::getP3OfLegCands(
+    const edm::Event& iEvent,
+    const edm::EDGetTokenT<trigger::TriggerFilterObjectWithRefs>& filterToken,
+    std::vector<math::XYZPoint>& p3s) {
   edm::Handle<trigger::TriggerFilterObjectWithRefs> filterOutput;
-  iEvent.getByToken(filterToken,filterOutput);
+  iEvent.getByToken(filterToken, filterOutput);
 
   //its easier on the if statement flow if I try everything at once, shouldnt add to timing
   std::vector<edm::Ref<reco::RecoEcalCandidateCollection> > phoCands;
-  filterOutput->getObjects(trigger::TriggerPhoton,phoCands);
+  filterOutput->getObjects(trigger::TriggerPhoton, phoCands);
   std::vector<edm::Ref<reco::RecoEcalCandidateCollection> > clusCands;
-  filterOutput->getObjects(trigger::TriggerCluster,clusCands);
+  filterOutput->getObjects(trigger::TriggerCluster, clusCands);
   std::vector<edm::Ref<reco::ElectronCollection> > eleCands;
-  filterOutput->getObjects(trigger::TriggerElectron,eleCands);
+  filterOutput->getObjects(trigger::TriggerElectron, eleCands);
   std::vector<edm::Ref<reco::CaloJetCollection> > jetCands;
-  filterOutput->getObjects(trigger::TriggerJet,jetCands);
+  filterOutput->getObjects(trigger::TriggerJet, jetCands);
 
-  if(!phoCands.empty()){ //its photons
-    for(auto & phoCand : phoCands){
+  if (!phoCands.empty()) {  //its photons
+    for (auto& phoCand : phoCands) {
       p3s.push_back(phoCand->superCluster()->position());
     }
-  }else if(!clusCands.empty()){ //try trigger cluster (should never be this, at the time of writing (17/1/11) this would indicate an error)
-    for(auto & clusCand : clusCands){
+  } else if (!clusCands
+                  .empty()) {  //try trigger cluster (should never be this, at the time of writing (17/1/11) this would indicate an error)
```

Comment:
here is the same

Reply:
of course we are not alone on earth
https://stackoverflow.com/questions/44731978/make-clang-format-ignore-comments-for-line-break

Reply:
the thread is from 2 years ago. It look "unfixable".

Reply:
@slava77 @VinInn we may manage it manually if/when really bothering...